API/ev3_lcd.c

```diff
@@ -1883,17 +1892,95 @@ int Ev3Printf(const char *fmt, ...)
 
 int Ev3Println(const char *fmt, ...)
 {
-	char *buffer;
-	vaprintf(buffer, "%s%s", "\n");

```

Comment:
Just a suggestion for the future, iIt would've been better if bugs were fixed in separate commits or even better squashed into the initial commit.